backend/src/platform/isolationEngine/environment.py

```
@@ -115,27 +182,28 @@ def seed_data_from_template(
                     seen.add(tbl)
 
-            conn.execute(text("SET session_replication_role = replica"))
+            self._ensure_constraints_deferrable(conn, target_schema)
+            conn.execute(text("SET CONSTRAINTS ALL DEFERRED"))
             try:
-                for tbl in ordered:
+                for tbl in ordered_tables:
                     conn.execute(
                         text(
                             f'INSERT INTO "{target_schema}"."{tbl}" '
                             f'SELECT * FROM "{template_schema}"."{tbl}"'
```

Ruff (0.14.5)

Possible SQL injection vector through string-based query construction

(S608)